### ISSUE TITLE
Luminosity ratio between keyboard focus and "May be later" and "Try it now" controls are less than 3:1

### ISSUE THREAD

**Issue by BharathShanker08** [microsoft/fluentui#15207]
User Experience:
Low vision users will face difficulty to identify the focus if Keyboard Focus is not properly visible and luminosity ratio between keyboard with respect to element is less than 3:1

Test Environment:
OS: Windows 10
Version: 2004(OS Build 19588.1000)
Matrix: Edge (Anaheim) + Narrator
Browser Version: 85.0.552.1 (Official build) dev (64-bit)
URL : https://developer.microsoft.com/en-us/fluentui#/controls/web/teachingbubble

Repro Steps:
Step 1: Open the above URL in Edge - Dev browser.
Step 2: "TeachingBubble" page will be displayed
Step 3: Navigate to "Show teaching bubble" control available in the "TeachingBubble Condensed" region and activate it
Step 4: Observe whether keyboard focus is clearly visible and Luminosity ratio is greater than 3:1

Actual :
Luminosity ratio between keyboard focus and "May be later" and "Try it now" controls are less than 3:1

Expected :
Luminosity ratio between keyboard focus and "May be later" and "Try it now" controls should be greater than or equal to 3:1

WCAG : [MAS 1.4.11 – Non-text Contrast](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/Doc.aspx?sourcedoc=%7B6CA3BF14-2635-434A-832C-28DD9A18B9FC%7D&file=MAS%201.4.11%20%E2%80%93%20Non-text%20Contrast.docx&action=default&mobileredirect=true)

**Comment by paulgildea** [comment]
@BharathShanker08 Thanks for the issue.

@miroslavstastny Try to determine if this is a recent regression or not.

@betrue-final-final  Any thoughts here? I looked at the Design Toolkit and there's no defined visual for focus inside a `TeachingBubble`
